app/views/search/_search_field.html.erb

```diff
@@ -1,5 +1,8 @@
 <% label_text = capture do %>
-  <h1>Search GOV.UK</h1>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Search GOV.UK",
+  heading_level: 1
```

Comment:
Dangling comma ftw:

```suggestion
  heading_level: 1,
```

Reply:
Ouu [I don't think we want a dangle](https://components.publishing.service.gov.uk/component-guide/heading)

Reply:
I'd say so - it gives a better diff when another parameter is added because only one line is changed instead of two.

Reply:
Oh I mean all the code examples on the gem don't have a dangle for the final param? 

I was thinking this was _intentional_ ruling? Partly as default [standardJS config](https://standardjs.com/rules.html) does the same (aware this is erb..)?

Reply:
Hmm, it's part of the [ruby linting configuration](https://github.com/alphagov/rubocop-govuk/blob/fdb474e9e17985da8b5d26c76f6a87573a4c53a6/config/style.yml#L23-L26) - maybe the linter isn't being run fully here. Whatever the reason for the linting not being run, it'd be good to have the comma here.